CONTRIBUTING.md

```diff
@@ -28,11 +29,14 @@ All contributors are expected to follow the [Rust Code of Conduct].
 
 ## Getting started
 
-High level approach:
+**Note: If this is your first time contributing to Clippy, you should
+first read the [Basics docs](doc/basics.md).**
```

Comment:
I bolded the whole thing to catch people's eyes (cc the rustc-dev-guide's getting started section).